### PR TITLE
ansible-gpii-version-updater

### PR DESCRIPTION
**Pull request by mrtyler**
For https://issues.gpii.net/browse/GPII-2542. This is the ansible role that runs the container built by https://github.com/gpii-ops/gpii-version-updater.

### CODE REVIEW

File: files/ssh_known_hosts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# NOTE: This file is managed by ansible-gpii-version-updater! Local changes will be discarded!
+#
+# Generated with e.g. "ssh-keyscan github.com"
+github.com ssh-rsa AAAAB3NzaC1yc2EAAAABIwAAAQEAq2A7hRGmdnm9tUDbO9IDSwBK6TbQa+PXYPCPy6rbTrTtw7PHkccKrpp0yVhp5HdEIcKr6pLlVDBfOLX9QUsyCOV0wzfjIJNlGEYsdlLJizHhbn2mUjvSAHQqZETYP81eFzLQNnPHt4EVVUh7VfDESU84KezmD5QlWpXLmvU31/yMf+Se8xhHTvKSCZIFImWwoG6mbUoWf9nzpIoaSjB+weqqUUmpaaasXVal72J+UX2B+2RPW3RcT0eOzQgqlJL3RKrTJvdsjE3JEAvGq3lGHSZXy28G3skua2SmVi/w4yCE6gbODqnTWlg7+wC604ydGXA8VJiS5ap43JXiUFFAaQ==
+gitlab.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCsj2bNKTBSpIYDEGk9KxsGh3mySTRgMtXL583qmBpzeQ+jqCMRgBqB98u3z++J1sKlXHWfM9dyhSevkMwSbhoR8XIq/U0tCNyokEi/ueaBMCvbcTHhO7FcwzY92WK4Yt0aGROY5qX2UKSeOvuP4D6TPqKF1onrSzH9bx9XUf2lEdWT/ia1NEKjunUqu1xOB/StKDHMoX4/OKyIzuS0q/T1zOATthvasJFoPrAjkohTyaDUz2LN5JoH839hViyEG82yB+MjcFV5MU3N1l1QL3cVUCh93xSaua1N85qivl+siMkPGbO5xR/En4iEY6K2XPASUEMaieWVNTRCtJ4S8H+9
+gitlab.com ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBFSMqzJeV9rUzU4kWitGjeR4PWSa29SPqJ1fVkhtj3Hw9xjLVXVYrU9QlYWrOLXBpQ6KWjbjTDTdDkoohFzgbEY=
+gitlab.com ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAfuCHKVTjquxvt6CM6tdG4SLp1Btn/nOeHHE5UOzRdf
```
Comment (gtirloni) [inline files/ssh_known_hosts:7]: We usually keep environment-dependent data in host_vars/group_vars. This information is not super sensitive but it'll go up in a public repo.

File: handlers/main.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+---
```
Comment (gtirloni) [inline handlers/main.yml]: File not used.

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,5 @@
+ansible
```
Comment (gtirloni) [inline requirements.txt]: With Ansible 2.4, molecule/testinfra/ansible-lint break. I have pinned other roles to 2.3.2.0

File: vars/main.yml [hunk 4]
```diff
@@ -0,0 +1 @@
+---
```
Comment (gtirloni) [inline vars/main.yml]: File not needed.

File: LICENSE [hunk 5]
```diff
@@ -0,0 +1,21 @@
+MIT License
```
Comment (gtirloni) [inline LICENSE]: May want to change this to the other license with specific disclaimers (as discussed somewhere else I can't remember).

File: defaults/main.yml [hunk 6]
```diff
@@ -0,0 +1 @@
+---
```
Comment (gtirloni) [inline defaults/main.yml]: File not needed.

**Review by gtirloni — COMMENTED** [review]
Just minor comments. Feel free to merge at your convenience.

File: files/ssh_known_hosts (hunk 1 above)
Comment (mrtyler) [reply files/ssh_known_hosts:7]: These are ssh public host keys for github.com and gitlab.com. They are neither environment-dependent nor sensitive information. So, I'm not sure what you're asking for. Can you elaborate?

File: handlers/main.yml (hunk 2 above)
Comment (mrtyler) [reply handlers/main.yml]: These come from molecule boilerplate, of course, but I'll delete them.

File: requirements.txt (hunk 3 above)
Comment (mrtyler) [reply requirements.txt]: Yeah, I saw your commit spree this morning :). Thanks for the heads up.

File: LICENSE (hunk 5 above)
Comment (mrtyler) [reply LICENSE]: Good catch.

File: files/ssh_known_hosts (hunk 1 above)
Comment (gtirloni) [reply files/ssh_known_hosts:7]: I missed they were for public hosts. Sorry. I was asking for the exact thing I mentioned considering I thought they were not public.

File: requirements.txt (hunk 3 above)
Comment (gtirloni) [reply requirements.txt]: molecule 1.25.1 has been released with fixes.

Comment (mrtyler) [reply requirements.txt]: With ansible 2.4.0.0 and molecule 1.25.1 I get this error:

```
tests/test_default.py:5: in <module>
    '.molecule/ansible_inventory').get_hosts('all')
../venv/lib/python2.7/site-packages/testinfra/utils/ansible_runner.py:150: in __init__
    self.variable_manager = ansible.vars.VariableManager()
E   AttributeError: 'module' object has no attribute 'VariableManager'
```

A quick google turns up nothing. I'm going to pin to the latest ansible 2.3.x for now.

**Pull request merged**